<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeEntry</class>
 <widget class="QWidget" name="ChangeEntry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>310</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="addLeftButton">
       <property name="text">
        <string>Add</string>
       </property>
       <property name="icon">
        <iconset theme="list-add">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="delLeftButton">
       <property name="text">
        <string>Remove</string>
       </property>
       <property name="icon">
        <iconset theme="list-remove">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="commonEdit"/>
     </item>
     <item>
      <widget class="QToolButton" name="delRightButton">
       <property name="text">
        <string>Remove</string>
       </property>
       <property name="icon">
        <iconset theme="list-remove">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="addRightButton">
       <property name="text">
        <string>Add</string>
       </property>
       <property name="icon">
        <iconset theme="list-add">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QGridLayout" name="attribLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="sheetLabel">
       <property name="text">
        <string>Sheet</string>
       </property>
       <property name="buddy">
        <cstring>sheetEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="sheetEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="areaLabel">
       <property name="text">
        <string>Area</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="areaCombo"/>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="afterLabel">
     <property name="text">
      <string>After</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="buddy">
      <cstring>afterView</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QListView" name="beforeView">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QListView" name="afterView">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="beforeLabel">
     <property name="text">
      <string>Before</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="buddy">
      <cstring>beforeView</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QPlainTextEdit" name="commentEdit">
     <property name="placeholderText">
      <string>Comments</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>sheetEdit</tabstop>
  <tabstop>beforeView</tabstop>
  <tabstop>afterView</tabstop>
  <tabstop>addLeftButton</tabstop>
  <tabstop>delLeftButton</tabstop>
  <tabstop>commonEdit</tabstop>
  <tabstop>delRightButton</tabstop>
  <tabstop>addRightButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>addLeftButton</sender>
   <signal>clicked()</signal>
   <receiver>ChangeEntry</receiver>
   <slot>addLeftClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>17</x>
     <y>287</y>
    </hint>
    <hint type="destinationlabel">
     <x>15</x>
     <y>360</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>delLeftButton</sender>
   <signal>clicked()</signal>
   <receiver>ChangeEntry</receiver>
   <slot>delLeftClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>43</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>53</x>
     <y>442</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>delRightButton</sender>
   <signal>clicked()</signal>
   <receiver>ChangeEntry</receiver>
   <slot>delRightClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>342</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>341</x>
     <y>428</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addRightButton</sender>
   <signal>clicked()</signal>
   <receiver>ChangeEntry</receiver>
   <slot>addRightClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>376</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel">
     <x>376</x>
     <y>443</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sheetEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>ChangeEntry</receiver>
   <slot>updateStatus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>302</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>637</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addLeftButton</sender>
   <signal>clicked()</signal>
   <receiver>ChangeEntry</receiver>
   <slot>updateStatus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>22</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>22</x>
     <y>393</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>delLeftButton</sender>
   <signal>clicked()</signal>
   <receiver>ChangeEntry</receiver>
   <slot>updateStatus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel">
     <x>78</x>
     <y>423</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>delRightButton</sender>
   <signal>clicked()</signal>
   <receiver>ChangeEntry</receiver>
   <slot>updateStatus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>348</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>471</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addRightButton</sender>
   <signal>clicked()</signal>
   <receiver>ChangeEntry</receiver>
   <slot>updateStatus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>378</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>403</x>
     <y>434</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>areaCombo</sender>
   <signal>activated(int)</signal>
   <receiver>ChangeEntry</receiver>
   <slot>updateStatus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>618</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>commentEdit</sender>
   <signal>textChanged()</signal>
   <receiver>ChangeEntry</receiver>
   <slot>updateStatus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>343</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>613</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addLeftClicked()</slot>
  <slot>addRightClicked()</slot>
  <slot>delLeftClicked()</slot>
  <slot>delRightClicked()</slot>
  <slot>updateStatus()</slot>
 </slots>
</ui>
